src/rulesets/oas2/functions/oas2UnusedDefinition.ts

```diff
@@ -0,0 +1,58 @@
+import { JsonPath } from '@stoplight/types';
+import { IFunction, IFunctionResult } from '../../../types';
+
+function* refIterator(obj: object, path: JsonPath): IterableIterator<[JsonPath, string]> {
```

Comment:
@P0lip I see that piece of code reused for the OAS3 to be created rule. However, I'm not sure where you'd prefer see this stored under the `/src` hierarchy. Any hint would be greatly appreciated.

Reply:
> @P0lip I foresee that piece of code reused for the OAS3 to be created rule. 

Yeah, would be great to reuse it.
I'd suggest placing it under:
```
src/rulesets/oas/functions/utils
```
Note, the `utils` directory does not exist yet, but feel free to create it.

Reply:
@P0lip Perfect.

Reply:
Fixed

Reply:
Eventually dropped.